src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs

```diff
@@ -10,10 +13,21 @@ namespace Dynamo.PackageManager.UI
     /// </summary>
     public partial class PackageManagerSearchView : Window
     {
+        private PackageManagerSearchViewModel PackageManagerSearchViewModel;
+
```

Comment:
Data member `PackageManagerSearchViewModel` isn't required, we can obtain its value from `this.DataContext`.